### ISSUE TITLE
different frame size

### ISSUE THREAD

**Issue by wangshiyao-1119** [mozilla/LPCNet#152]
Has anyone tried different sample rates and different frame lengths? 
For example, 22.5ms. With different frame lengths:
#define FRAME_SIZE_5MS (2) 
#define OVERLAP_SIZE_5MS (2)   
#define TRAINING_OFFSET_5MS (1)   
#define WINDOW_SIZE_5MS (FRAME_SIZE_5MS + OVERLAP_SIZE_5MS)

Should these parameters be changed?

**Comment by xiaoyangnihao** [comment]
these parameters may not be changed.

**Comment by wangshiyao-1119** [comment]
> these parameters may not be changed.

Thank you for your reply！
Have you tried increasing the frame length? I increased the frame length to 22.5ms, and the sound quality of the synthesized speech degraded greatly

**Comment by xiaoyangnihao** [comment]
You should change NB_BANDS, LPCNET_FRAME_SIZE and some other params.

**Comment by wangshiyao-1119** [comment]
> You should change NB_BANDS, LPCNET_FRAME_SIZE and some other params.

Yes, I changed these parameters accordingly, but the quality is still degraded.Do you think it has something to do with the network part？
